DEV_ENVIRONMENT.md

```diff
@@ -0,0 +1,32 @@
+# Developer Environment Setup
+
+## Desktop app and Flipper Plugins (Javascript)
+
+The flipper desktop source uses Flow and ESLint. Feel free to use your preferred setup, but this is our recommended approach:
+
+**Editor**: Visual Studio Code
+
+**Installed Extensions**:
+ * ESLint
+ * Flow Language Support
+ 
+**Settings**:
+```json
+{
+    "flow.useNPMPackagedFlow": true,
+    "javascript.validate.enable": false,
+    "eslint.autoFixOnSave": true
+}
+```
+
+## Android plugins and SDK (Java)
+
+**Editor**: Android Studio
+
+## iOS plugins and SDK (Objective-C)
+
+**Editor**: XCode
+
+## Cross-platform SDK (Used by Android and iOS SDK) (C++)
+
+**Editor**: XCode, Visual Studio Code, Vim...
```

Comment:
I think Android Studio is actually not terrible here?

Reply:
Added.